src/lib/arrays.js

```diff
@@ -0,0 +1,123 @@
+const getNthElement = (index, array) => {
+  const index2 = index - array.length;
+
+  if (index >= array.length) {
+    return array[index2];
+  }
+  return array[index];
+};
+
+const arrayToCSVString = array => {
+  return array.toString();
+};
+
+const csvStringToArray = string => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  array.push(element);
+};
+
+const addToArray2 = (element, array) => {
+  return array.concat([element]);
+};
+
+const removeNthElement = (index, array) => {
+  return array.splice(index, 1);
+};
+
+const numbersToStrings = numbers => {
+  const numberString = numbers.map(number => {
+    return number.toString();
+  });
+  return numberString;
+};
+
+const uppercaseWordsInArray = strings => {
+  const uppercaseWords = strings.map(word => {
+    return word.toUpperCase();
+  });
+  return uppercaseWords;
+};
+
+const reverseWordsInArray = strings => {
+  const stringsToArrays = strings.map(element => {
```

Comment:
This could be refactored into one line.

What you do is, split the array, then reverse it and the join back together. You could use the split, reverse and join array methods:
````
  return strings.map(str =>
    str
      .split('')
      .reverse()
      .join('')
`````